### DISCUSSION TITLE
Add ID to a button

### DISCUSSION THREAD

**Discussion by IvyMasterDiego** [squidfunk/mkdocs-material#7077] [Q&A] [answered]
It is possible to add an id to a button in this way:

Example: 

```
[click](/docs/javascript/test.js){.md-button .md-button--primary #buttonclick}
```
And if this is not the way, how could I add this id to be able to integrate it with js.

**Comment by IvyMasterDiego** [comment]
i try my self so no need answer tenkiu

**Reply by kamilkrzyskow** [reply, marked as answer]
Yes, that's the use-case for attribute lists:
- https://python-markdown.github.io/extensions/attr_list/
- https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists
